### PR TITLE
add ArgusObserver/wagu

### PR DESCRIPTION
**Pull request by amousa11**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,6 +130,7 @@ See also [Friends of Rust](https://www.rust-lang.org/friends.html)(organizations
 * [parity-ethereum](https://github.com/paritytech/parity-ethereum) — Fast, light, and robust Ethereum client
 * [parity-bitcoin](https://github.com/paritytech/parity-bitcoin) — The Parity Bitcoin client [<img src="https://api.travis-ci.org/paritytech/parity-bitcoin.svg?branch=master">](https://travis-ci.com/paritytech/parity-bitcoin)
 * [parity-bridge](https://github.com/paritytech/parity-bridge) — Bridge between any two ethereum-based networks
+* [ArgusObserver/wagu](https://github.com/ArgusObserver/wagu) [[wagu](https://crates.io/crates/wagu)] — Generate a wallet for any cryptocurrency [<img src="https://api.travis-ci.com/ArgusObserver/wagu.svg?branch=master">](https://api.travis-ci.com/ArgusObserver/wagu.svg?branch=master)
```
Comment (luciusmagn) [inline README.md]: > [[wagu](https://crates.io/crates/wagu)]

Are you sure you want the extra [] around the link to crates.io?

Comment (amousa11) [reply README.md]: I think it’s fine since this is how the contribution guide recommends it.

Comment (luciusmagn) [reply README.md]: you are right, it just looked to me weird since it has been quite a long time since I saw any entry that did that, sorry :sweat_smile:

**Comment by luciusmagn** [comment]
Thanks for the addition, looks good!

**Pull request merged**